packages/www/components/Dashboard/StreamDetails/SessionsTable/helpers.tsx

```diff
@@ -138,7 +138,7 @@ export const rowsPageFromState = async (
             ) : (
               <Box>—</Box>
             ),
-          mp4Url: stream.recordingUrl ? stream.recordingUrl : undefined,
+          mp4Url: stream.mp4Url ? stream.mp4Url : stream.recordingUrl,
```

Comment:
**nit:** by definition this renders the same result
```suggestion
          mp4Url: stream.mp4Url ?? stream.recordingUrl,
```

Reply:
Althoooough, do we want to show an HLS URL on the download button? I think we probably just don't show anything
```suggestion
          mp4Url: stream.mp4Url,
```

Reply:
I think we need to keep it that way for the sake of backwards compatibility. This `mp4Url` is passed to `makeMP4Url()` and then mp4 url is created from hlsUrl.

Reply:
Ahh I see! Hadn't got that part. Sounds good then 😄